content/remote-development/getting-started.md

```diff
@@ -36,33 +36,40 @@ brew install continuouspipe/tools/cp-remote
 Otherwise you can install it manually via:
 
 ```
-sudo curl https://continuouspipe.github.io/remote-environment-client/0.1.0/darwin-amd64.gz > cp-remote.gz
-gzip -d cp-remote.gz;
+sudo curl https://continuouspipe.github.io/remote-environment-client/latest/darwin-amd64/cp-remote.tar.gz > cp-remote.tar.gz
+tar -xzvf cp-remote.tar.gz;
```

Comment:
I think when you are bundling tar options, the best practice is to drop the preceding dash.